package-lock.json

```diff
@@ -3044,9 +3044,9 @@
       }
     },
     "node_modules/aws-sdk": {
-      "version": "2.1343.0",
-      "resolved": "https://registry.npmjs.org/aws-sdk/-/aws-sdk-2.1343.0.tgz",
-      "integrity": "sha512-p9+bgEeD2HxFMLAl5IyqvxfmFzuzSwyvoKfEaGL3+0Vndztv7TvCbHfKNH6olY9DjyLrq9f34fJ98owGD8sHnQ==",
+      "version": "2.1356.0",
```

Comment:
The new v2 may introduce an upgrade warning which is kind of noisy for tests, do we want to supress that?

Reply:
Oh yeah, it might be a good idea. Let's do the same as in Serverless then

Reply:
I just checked a random build and it seems there is no noisy warning about the aws-sdk v2 🤷🏼 
https://github.com/serverless-heaven/serverless-webpack/actions/runs/4678437131/jobs/8287167794?pr=1437

Reply:
Interesting, serverless/serverless#11753 suggests the warning appears since `aws-sdk@^2.1315` and 2.1356.0 should have it too. But if the warning is not there we can leave it as-is.